ros2param/test/test_verb_dump.py

```diff
@@ -158,6 +158,8 @@ def setUp(self):
                     services = node.get_service_names_and_types_by_node(TEST_NODE, TEST_NAMESPACE)
                 except rclpy.node.NodeNameNonExistentError:
                     continue
+                except ConnectionRefusedError:
+                    continue
```

Comment:
this can give a chance to go to the next loop, and that is going to fall back to `DirectNode` if the exception reason is the XMLRPC connection provided by daemon server. although this does not fix the root cause of `ConnectionRefusedError` via XMLRPC, it can be more robust to avoid being affected by ros2daemon instability.